recipes/r-rpostgres/meta.yaml

```diff
@@ -52,7 +52,7 @@ requirements:
   run:
     - r-base
     - {{native}}gcc-libs         # [win]
-    - libgfortran                # [not win]
+    - libgfortran-ng                # [not win]
```

Comment:
please use this instead if fortran is really needed `{{ compiler('fortran') }}  # [not win]` in your build section.

Reply:
@bgruening RPostgres doesn't need a fortran compiler itself, but PostgreSQL does. So I've removed the `libgfortran` dependency and added `postgresql` to the list of run-time dependencies. Now the builds are successful without an explicit `libgfortran` dependency.